### PR TITLE
Include minimum 6 month lockup in commitment date calculation

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

File: src/components/Modal/TACoCommitmentModal/index.tsx [hunk 1]
```diff
@@ -38,7 +38,8 @@ export type TACoCommitProps = BaseModalProps & {
 
 const getEndDate = (durationInSeconds: number) => {
   const endDate = new Date()
-  endDate.setSeconds(endDate.getSeconds() + durationInSeconds)
+  // extra 15724800 seconds for the minimum 6 month lock up
```
Comment (derekpierre) [inline src/components/Modal/TACoCommitmentModal/index.tsx:41]:
```suggestion
  // extra 15724800 seconds for the 6-month deauthorization
```

**Review by derekpierre — APPROVED** [review]
🎸 - one suggestion.

**Review by arjunhassard — APPROVED** [review]
Thanks for the quick fix, would have been very confusing!!

**Review by cygnusv — APPROVED** [review]

**Review by derekpierre — APPROVED** [review]

**Comment by theref** [comment]
This backport isn't required, the change is on `v1.14.1`

**Comment by michalsmiarowski** [comment]
Just adding a note here that this backport isn't required because we've created a separate branch for `v1.14.1`, where this change is applied (see Backport of 0a26de3fabcb96ca2f58a66459e81c0616a55018).

**Pull request closed without merging**